src/ppmdu/pmd2/pmd2.hpp

```diff
@@ -43,11 +43,7 @@ namespace pmd2
     const std::string CommonXMLGameVersionAttrStr = "gameVersion";
     const std::string CommonXMLGameRegionAttrStr  = "gameRegion";
     const std::string CommonXMLToolVersionAttrStr = "libVersion";
-
-    /*
-        Current version string. Represents the version of the PPMDU library used.
-    */
-    const std::string PMD2ToolsetVersion = "0.2.0"; //! #TODO: Replace with something that takes care of incrementing this for us!
```

Comment:
Need to keep PMD2ToolsetVersion defined as an external constant to fix a build error